src/bin/wallet.rs

```diff
@@ -142,6 +162,27 @@ fn parse_args() -> Result<WalletConfig, Box<error::Error>> {
                         .help("The transaction signature to confirm"),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("transfer")
+                .about("Transfer raw transaction from file or stdin")
+                .arg(
+                    Arg::with_name("userdata_path")
```

Comment:
if we allow `transfer` to read from stdin if `userdata-path` is not supplied, like `build` writes to stdout if `outfile` is not supplied, then we can get rid of the `serial_userdata` arg I think

Reply:
Yeah, that's an option. I did it this way to use clap's built in handling of missing parameters -- ie. clap requires either the `-f` option OR data. Saves having to write that error handling (if `transfer` is run without either). Probably about the same amount of code either way.